src/api/db/models/Project.ts

```diff
@@ -334,6 +367,49 @@ export class ProjectModel {
     }
   }
 
+  static async removeCameraFromViews(
```

Comment:
Probably a bit convoluted way to do it, open to suggestions

Reply:
I think this works! No ideas jumping to mind for streamlining it. I think all of those steps are necessary.